code/src/vs/platform/extensionManagement/common/extensionsScannerService.ts

```diff
@@ -412,6 +412,17 @@ class ExtensionsScanner extends Disposable {
 		super();
 	}
 
+	private readonly excludedExtensions = [
+		'vscode-api-tests',
+		'vscode-colorize-tests',
+		'vscode-test-resolver',
+		'ms-vscode.node-debug',
+		'ms-vscode.node-debug2',
+		'vscode-notebook-tests',
+		'vscode-custom-editor-tests',
```

Comment:
why these one are excluded ?

Reply:
The list comes from https://github.com/che-incubator/che-code/blob/main/code/build/lib/extensions.ts#L282-L290. I thought that it was added by our team and since it doesn't work for `github-authentication` it doesn't work for others, so I decided to add those extensions as well. Am I correct?

Reply:
it's added by upstream vscode so I would just add there github one

Reply:
done